### PR TITLE
Add page.kramo.Cartridges

### PR DESCRIPTION
**Pull request by kra-mo**
hu.kramo.Cartridges is being renamed https://github.com/flathub/flathub/pull/5125

### CODE REVIEW

File: flatpak_builder_lint/staticfiles/exceptions.json [hunk 1]
```diff
@@ -2420,5 +2420,9 @@
     },
     "com.core447.StreamController": {
         "finish-args-flatpak-spawn-access": "required to allow plugins to open apps on the host"
+    },
+	"page.kramo.Cartridges": {
+    	"finish-args-flatpak-spawn-access": "required to launch games on the host",
+        "finish-args-unnecessary-xdg-data-access": "access to xdg-data/flatpak/app and exports is needed to import games installed as user flatpaks"
```
Comment (bbhtt) [inline flatpak_builder_lint/staticfiles/exceptions.json:2426]:
```suggestion
    	"finish-args-flatpak-spawn-access": "required to launch games on the host",
      "finish-args-unnecessary-xdg-data-access": "access to xdg-data/flatpak/app and exports is needed to import games installed as user flatpaks"
```

Comment (bbhtt) [inline flatpak_builder_lint/staticfiles/exceptions.json:2426]: Slight indentation problem here

image

Comment (kra-mo) [reply flatpak_builder_lint/staticfiles/exceptions.json]: Alright, resolved

**Comment by bbhtt** [comment]
You have a merge conflict on the file.

**Comment by kra-mo** [comment]
Alright, fixed

**Pull request merged**